eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyRemote.java

```diff
@@ -73,10 +78,8 @@ public Input get(final String name) throws MalformedURLException {
     }
 
     @Override
-    public boolean contains(final String name) throws IOException {
-        final int code = ((HttpURLConnection) this.template.value(name).openConnection())
-            .getResponseCode();
-        return code >= HttpURLConnection.HTTP_OK && code < HttpURLConnection.HTTP_BAD_REQUEST;
+    public boolean contains(final String name) {
```

Comment:
@volodya-lombrozo I don't think it's reasonable to rely only on the cache file. It may be broken, absent, not complete. I think it's better to check the file first and then, if the element is absent in the file, we make an HTTP connection to double check. I would actually introduce a new `Oy*` object for this.

Reply:
`OyIndexed`